### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by dylangrimes1**
review of code of conduct

### CODE REVIEW

File: codeofconduct [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<title>Code Of Conduct</title>
+</head>
+<body>
+<h1>Code Of Conduct</h1>
+<p>The Code of Conduct is a very important set of rules that all teams need to follow, some of these rules may apply more to one team than another, however every rule will be important and must be followed by everyone. These rules are put in place to ensure that everyone acts appropriately in the workplace. Every rule below must be followed, and failure to do so will not be tolerated. These rules may vary in severity and consequences</p>
+<h3>Honesty</h3>
+<p>1.	No member of staff will lie to their employer when it comes to professional topics such as deadlines, progress and where they have been during work hours.</p>
+<br>
+<h3>Deadlines</h3>
+<p>2.	All deadlines must be completed in time, If there is any reason preventing a staff member completing a deadline, they must talk to their employer in advance to discuss extensions.</p>
```
Comment (DannyENL) [inline codeofconduct:13]: I would change "in time, " to "on time. "

File: codeofconduct [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+<br>
+<h3>Anti-Harassment</h3>
+<p>3.	Harassment towards any member of staff will not be tolerated, the workplace environment must be safe for all staff members, physical and verbal harassment is taken very seriously</p>
+<br>
+<h3>Anti-Discrimination</h3>
+<p>4.	Discrimination, such as sexism, racism, ableism or homophobia will be taken very seriously and will be looked at individually depending on what happened</p>
+<br>
+<h3>Virtual Harrasment/discrimination</h3>
+<p>5.	The use of computers to harass or discriminate against another person virtually or anonymously will be investigated </p>
```
Comment (DannyENL) [inline codeofconduct:22]: Missing full stop

File: codeofconduct [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+<br>
+<h3>Misinformation</h3>
+<p>6.	Spreading any misinformation deliberately is very dangerous and can lead to legal trouble for the individual or for the entire company</p>
+<br>
+<h3>Attendance</h3>
+<p>7.	Every member of staff must show up to their scheduled hours, if for whatever reason someone cannot attend, they must discuss that with their employer in advance</p>
```
Comment (DannyENL) [inline codeofconduct:28]: Missing full stop

File: codeofconduct [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+<h1>Code Of Conduct</h1>
+<p>The Code of Conduct is a very important set of rules that all teams need to follow, some of these rules may apply more to one team than another, however every rule will be important and must be followed by everyone. These rules are put in place to ensure that everyone acts appropriately in the workplace. Every rule below must be followed, and failure to do so will not be tolerated. These rules may vary in severity and consequences</p>
+<h3>Honesty</h3>
+<p>1.	No member of staff will lie to their employer when it comes to professional topics such as deadlines, progress and where they have been during work hours.</p>
+<br>
+<h3>Deadlines</h3>
+<p>2.	All deadlines must be completed in time, If there is any reason preventing a staff member completing a deadline, they must talk to their employer in advance to discuss extensions.</p>
+<br>
+<h3>Anti-Harassment</h3>
+<p>3.	Harassment towards any member of staff will not be tolerated, the workplace environment must be safe for all staff members, physical and verbal harassment is taken very seriously</p>
```
Comment (DannyENL) [inline codeofconduct:16]: I would change "not be tolerated, the workplace environment", to "not be tolerated. The workplace environment". I would also change "members, physical" to "members; physical".

File: codeofconduct [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+<br>
+<h3>Anti-Discrimination</h3>
+<p>4.	Discrimination, such as sexism, racism, ableism or homophobia will be taken very seriously and will be looked at individually depending on what happened</p>
```
Comment (DannyENL) [inline codeofconduct:19]: Missing full stop

**Review by DannyENL — CHANGES_REQUESTED** [review]
The file doesn't have a html extension, which I think it'll need in order to be part of the website.
A few of the sentences don't end in full stops and certain sentences seem to run on for a while. I put in some suggestions for where a long sentence could be split up to make it a bit easier to read.

**Comment by ProtoRedux** [comment]
Gonna close this pull request cause the changes are included in the latest code of conduct pull request.

**Pull request closed without merging**